webui/react/src/services/types.ts

```diff
@@ -265,3 +265,8 @@ export interface GetJobsResponse extends Api.V1GetJobsResponse {
 export interface GetJobQStatsParams extends FetchOptions {
   resourcePools?: string[];
 }
+
+export interface SetUserPasswordParams {
+  password: string,
+  username: string
```

Comment:
nit: should probably end these lines with semicolons instead of commas (though both are valid)